metricbeat/docs/modules/docker.asciidoc

```diff
@@ -58,6 +58,11 @@ metricbeat.modules:
   # If set to true, replace dots in labels with `_`.
   #labels.dedot: false
 
+  # Skip metrics certain device major numbers in docker/diskio. 
```

Comment:
```suggestion
  # Skip metrics for certain device major numbers in docker/diskio. 
```
Or something like that.

I think it would be helpful to mention what the default values are, and that (presumably) setting `skip_major: []` would remove them.

Reply:
Ah, good point.